rl_algorithms/__init__.py

```diff
@@ -1,8 +1,10 @@
+from rl_algorithms.common.networks.backbones.cnn import CNN
+from rl_algorithms.common.networks.backbones.resnet import ResNet
```

Comment:
should change `from rl_algorithms.common.networks.backbones import CNN, ResNet` using `__init__.py`